.github/workflows/release.yml

```diff
@@ -127,22 +127,28 @@ jobs:
 
       - name: Get PR Description
         id: get_pr_description
```

Comment:
- It might be beneficial to include error handling in case the PR description retrieval fails for any reason.